packages/plugin-loansresearch-ui/src/components/form/Loans.tsx

```
@@ -0,0 +1,146 @@
+  const renderIncomeForm = () => {
+    const attachment =
+      (loansResearch.incomes &&
+        extractAttachment(loansResearch.incomes.files)) ||
+      [];
+
+    return (
+      <>
+        {incomes.map((income, index) => {
+          return (
+            <>
+              <FormGroup>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)